applications/nrf_desktop/doc/ble_bond.rst

```diff
@@ -71,11 +71,11 @@ The following diagram shows states and transitions between these states after th
    nRF Desktop Bluetooth LE bond module state diagram (click to enlarge)
 
 .. note::
-  The diagram does not present the states related to module going into standby (:c:enum:`STATE_STANDBY`, :c:enum:`STATE_DISABLED_STANDBY`, :c:enum:`STATE_DONGLE_CONN_STANDBY`).
+  The diagram does not present the states related to module going into standby (:c:enum:`STATE_STANDBY`, :c:enum:`STATE_DISABLED_STANDBY`, :c:enum:`STATE_DONGLE_STANDBY`).
   For more information about the standby states, see `Standby states`_.
 
 Receiving ``click_event`` with a click type that is not included in the schematic will result in cancelling the ongoing operation and returning to :c:enumerator:`STATE_IDLE`.
-This does not apply to :c:enumerator:`STATE_DONGLE_CONN`.
+This does not apply to :c:enumerator:`STATE_DONGLE`.
```

Comment:
Actually this may apply to the dongle states if one of the new Kconfig options is enabled

Reply:
Same as with diagram, will introduce changes when Grzegorz is available.